src/components/EditImageModal/index.jsx

```diff
@@ -19,23 +22,31 @@ export default class EditImageModal extends React.Component {
       imageDescription: '',
       imageDimensions: {},
       isImageDecorative: false,
+      isStatusAlertOpen: false,
       imageLoading: false,
       imageSource: '',
       imageStyle: '',
       open: false,
+      validationMessages: {},
     };
 
+    this.getStatusAlert = this.getStatusAlert.bind(this);
+    this.getStatusAlertDialog = this.getStatusAlertDialog.bind(this);
     this.handleOpenModal = this.handleOpenModal.bind(this);
     this.onConstrainProportionsClick = this.onConstrainProportionsClick.bind(this);
     this.onImageDescriptionBlur = this.onImageDescriptionBlur.bind(this);
     this.onInsertImageButtonClick = this.onInsertImageButtonClick.bind(this);
     this.onImageIsDecorativeClick = this.onImageIsDecorativeClick.bind(this);
     this.onImageLoad = this.onImageLoad.bind(this);
     this.onImageSourceBlur = this.onImageSourceBlur.bind(this);
+    this.onStatusAlertClose = this.onStatusAlertClose.bind(this);
+    this.validateImageDescription = this.validateImageDescription.bind(this);
 
-    this.formRef = {};
+    this.formRef = null;
```

Comment:
I think the initial value should be null, per [this](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs) React documentation.